observer/stream.go

```diff
@@ -100,7 +101,10 @@ func (s *Stream) loadBlock(c chan<- *blockatlas.Block, num int64) {
 	// Not strictly correct nor avoids race conditions
 	// But good enough
 	newNum := atomic.AddInt64(&s.blockNumber, 1)
-	err = s.Tracker.SetBlockNumber(s.coin, newNum)
```

Comment:
What happened here? It's removed forever.

Reply:
We need to restore this line after the if-block

Reply:
I only commited Eddys suggestion. Will review

Reply:
Done

Reply:
@akinmail you haven't pushed it seems